### PR TITLE
[skip ci]bugfix: change release_to_atlas.py to fit new Atlas class

### PR DESCRIPTION
**Pull request by changev**
class ```Atlas``` has been changed but ```release_to_atlas.py``` need to be modified too.

@panpan0000 @PengTian0 

test results: http://10.240.19.21/job/BuildRelease/job/Release/job/Release/118/

### CODE REVIEW

**Comment by PengTian0** [comment]
:+1:

File: build-release-tools/application/release_to_atlas.py [hunk 1]
```diff
@@ -9,20 +9,18 @@
...
+    --atlas-creds rackhd:****** \
     --atlas-name rackhd \
-    --atlas-token ****** \
     --atlas-version 1.2.3 \
     --is-release true
 
 The required parameters:
     build-directory: A directory where box files laid in.
-    atlas-token: atlas access token.
+    atlas-creds: value or env var name of rackhd:****.
```
Comment (panpan0000) [inline build-release-tools/application/release_to_atlas.py]: Can you please comment in code for the syntax ?
say, USER:API-TOKEN ( or password ? )
 then it's a 👍

File: build-release-tools/application/release_to_atlas.py [hunk 2]
```diff
@@ -9,20 +9,18 @@
...
+    --atlas-creds rackhd:****** \
     --atlas-name rackhd \
-    --atlas-token ****** \
     --atlas-version 1.2.3 \
     --is-release true
 
 The required parameters:
     build-directory: A directory where box files laid in.
-    atlas-token: atlas access token.
+    atlas-creds: value or env var name of atlas creds, format is atlas_username:atlas_token.
```
Comment (panpan0000) [inline build-release-tools/application/release_to_atlas.py:19]: Can you please comment in code for the syntax ?
say, USER:API-TOKEN ( or password ? )
 then it's a 👍

**Review by panpan0000 — APPROVED** [review]
👍

**Review by anhou — APPROVED** [review]

**Pull request merged**